tutorials/tutorial_workflows_v2.md

```diff
@@ -0,0 +1,78 @@
+---
+permalink: /tutorial-workflows-v2
+title: Interacting with Workflows V2
+layout: default
+section_title: Guides and Tutorials
+---
+
+## Introduction
+
+(boilerplate explanation of the tool like V1's)
+
+---
+
+## Hierarchy of Workflows
+
+There are 3 main levels of a workflow: template, preset and instance.
+
+![API Record Diagram]({{ site.baseurl }}/assets/guides/workflows-v2-api-records-diagram.png)
```

Comment:
I want to show the core resources that we have APIs for. Also the ones that we'd most likely make public first.